### ISSUE TITLE
MQTT Communicator not consistently publishing

### ISSUE THREAD

**Issue by rkoshak** [rkoshak/sensorReporter#111]
I noticed that for a couple of weeks I've not been getting updates on my door status changes and only one of three installs are publishing heartbeat messages. 

The communicator itself is working because I do get that first message published for each sensor when sensorReporter first starts up. That also shows that the sensors are configured to use that communicator. 

But after that point nothing gets published. From looking at the logs it is detecting that the doors are opening and closing, it's just not publishing that fact.

And the heartbeat sensor is only publishing at all on two of my three instances with one simply never publishing at all. 

In summary. 
- govee sensor 
- Heartbeat works on two out of three deployments
- RPI GPIO Sensor only publishes during startup and never again, but it does sense the state changes on the pins

I'm still researching but configs seem to be the same for the heartbeat stuff so it's strange that it doesn't work.

**Comment by rkoshak** [comment]
The GPIO Sensor problems may be my failure to keep up with some changes to configs. The heartbeat problem is still a mystery and investigations continue...

**Comment by rkoshak** [comment]
Yea, ID-10-T problems. It's all working now. I was inconsistent in how I updated my configs with the changes that have been made over time.
